src/Http/Http.Abstractions/test/HttpResponseWritingExtensionsTests.cs

```diff
@@ -32,33 +30,6 @@ public async Task WritingText_MultipleWrites()
             Assert.Equal(22, context.Response.Body.Length);
         }
 
-        [Theory]
-        [MemberData(nameof(Encodings))]
-        public async Task WritingTextThatRequiresMultipleSegmentsWorks(Encoding encoding)
```

Comment:
Why did this get removed?

Reply:
StreamPipeWriter doesn't exist anymore. I'll try rewriting this test to make it work with our current wrappers.